www/servers.html

```diff
@@ -51,7 +51,7 @@ <h3>Roubaix-fr.pirateirc.net</h3>
       <p><strong>location</strong>: Roubaix, France</br>
       <strong>party</strong>: Piraattipuolue Finland</br>
       <strong>staff</strong>: <a href="mailto:scoffa@pirateirc.net">Scoffa</a>, <a href="mailto:mikaela.suomalainen@piraattipuolue.fi">Mikaela</a>, <a href="mailto:ari-martti.hopiavuori@piraattipuolue.fi">AriMartti</a></br>
-      <strong>features</strong>: IPv4, SSL (commercial / verifiable), SASL</br>
+      <strong>features</strong>: IPv4, SSL (commercial / verifiable), SASL,</br> &emsp;<strong>Tor:</strong>&nbsp<code>tll4bxf546kzf6iv4n2m4pbbjnifrfewe3kcritva2tuuuiowygx2cqd.onion</code>,</br> &emsp;<strong>Yggdrasil:</strong>&nbsp<code>201:677d:1809:c5eb:202a:d39:e598:305</code></br>
```

Comment:
I need feedback about this formatting, it looks OKish on my local Firefox, but it should probably be done better?